src/models/account.model.js

```diff
@@ -52,7 +52,7 @@ module.exports = class Account extends BaseModel {
     try {
       // Map the Account to the corresponding Dynamics schema Account object
       const dataObject = {
-        defra_companyhouseid: Utilities.stripWhitespace(this.companyNumber).toUpperCase(),
+        defra_companyhouseid: Utilities.stripWhitespace(this.companyNumber).toUpperCase().replace(/ /g, ''),
```

Comment:
Do you need the call to .replace(...)? I think that the stripWhitespace() method will have already done?